src/schema/rules/sidecars/continuous.yaml

```diff
@@ -19,7 +19,51 @@ PhysioHardware:
   selectors:
     - suffix == "physio"
   fields:
+    DeviceSerialNumber: recommended
     Manufacturer: recommended
     ManufacturersModelName: recommended
     SoftwareVersions: recommended
-    DeviceSerialNumber: recommended
+
+PhysioTypeRecommended:
+  selectors:
+    - suffix == "physio"
+  fields:
+    PhysioType: recommended
+
+PhysioTypeRequired:
+  selectors:
+    - suffix == "physio"
+  fields:
+    PhysioType: required
+
+PhysioEvents:
+  selectors:
+    - suffix == "physioevents"
+  fields:
+    Columns: required
+    Description: recommended
+    ForeignIndexColumn: recommended
+
+# Eye-tracking
+EyeTrack:
+  selectors:
+    - suffix == "physio"
+  fields:
+    EnvironmentCoordinates: required
```

Comment:
This one is very problematic if REQUIRED, what happens when "SampleCoordinateSystem" is not "gaze-on-screen"?

Does this make sense for "eye-in-head" or "gaze-in-world"?